core/templates/dev/head/services/navigation-bars-backend-api.service.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to retrieve notifications from the
+ * backend.
+ */
+
+// eslint-disable-next-line max-len
+require('components/common-layout-directives/navigation-bars/navigation-bars.constants.ajs.ts');
```

Comment:
I'm not sure about the name of this service if it's expected to retrieve the notification, then I would suggest the name to be `NotificationBackendApiService`. (OR move the function to UserDataBackendApiServices, but I think NotificationBackendApiService would be better.)

@bansalnitish @seanlip, any thoughts on services name (in general)?

Reply:
I agree with @DubeySandeep. NotificationsBackendApiService seems like the right "sort" of name to me, since the backend API service should talk about what it's fetching rather than what uses it.